src/Implementation/InputDefinition/BoolDefinition.php

```diff
@@ -5,8 +5,20 @@
 
 class BoolDefinition extends InputDefinition
 {
+    /**
+     * @param string $name
+     */
     public function __construct($name)
     {
         parent::__construct($name, false, true, '', false);
     }
+
+    /**
+     * @param mixed $input
+     * @return boolean
+     */
+    public function processValue($input)
+    {
+        return true;
```

Comment:
nothing happens here

Reply:
the point is, that for a boolean to be true the key just needs to exist in the options array. Since that was already tested bfore this, there is sadly no other condition to check.

Reply:
just noticing the access wasn't meant to bee public, I guess that is more than just a bit confusing